no-computer.md

```diff
@@ -1,5 +1,4 @@
-# No computer project
-
+# No computer project - Recipe Book
```

Comment:
I found it funny how your no computer project is (kind of) one of my DSL projects.